project_issue_timesheet_time_control/models/account_analytic_line.py

```diff
@@ -12,25 +12,24 @@ class AccountAnalyticLine(models.Model):
         related='issue_id.stage_id.closed', readonly=True,
     )
 
-    @api.onchange('account_id')
-    def onchange_account_id_project_issue_timesheet_time_control(self):
-        if self.issue_id.analytic_account_id != self.account_id:
-            self.issue_id = False
-        domain = {'issue_id': []}
-        if self.account_id:
-            project = self.env['project.project'].sudo().search(
-                [('analytic_account_id', '=', self.account_id.id)], limit=1,
-            )
-            domain = {'issue_id': [('project_id', '=', project.id),
-                                   ('stage_id.closed', '=', False)]}
-        return {'domain': domain}
+    @api.onchange('project_id')
+    def onchange_project_id_project_issue_timesheet_time_control(self):
+        res = {}
+        if self.project_id:
+            # self.issue_id = False
```

Comment:
That line is commented because when you create a new line and select an issue, the onchange_issue_id_project_issue_timesheet_time_control() method fill the project field, and then it triggers the onchange_project_id_project_issue_timesheet_time_control() erasing the issue field and forcing to select again the issue.